### ISSUE TITLE
output too many logs to `/var/log/syslog` and `/var/log/daemon.log` when the mouse moving

### ISSUE THREAD

**Issue by kt286** [linuxdeepin/developer-center#3241]
### SUMMARY | 问题概要

output too many logs to `/var/log/syslog` and `/var/log/daemon.log` when the mouse moving

### OBSERVED RESULT | 观察到的结果

![image](https://user-images.githubusercontent.com/10684972/173192226-83a35a3f-ece6-43bd-a93a-a4aaf62e5e52.png)

the content like this:
```
2022-06-11 22:30:00 cloud dde-system-daemon[2349]:  libinput error: client bug: Invalid event type 400 passed to libinput_event_pointer_get_axis_source()
2022-06-11 22:30:00 cloud dde-system-daemon[2349]:  libinput error: client bug: Invalid event type 400 passed to libinput_event_pointer_get_axis_value()
```

### SOFTWARE/OS VERSIONS | 软件/系统版本信息
dde-daemon                                    5.14.44-1+kt286 (used this patch https://github.com/linuxdeepin/dde-daemon/commit/04c900d31414b3ea8f3f69da8e1542ce7b7d9e59 )
libinput-bin                                  1.12.6.1-1+deepin
libinput-dev:amd64                            1.12.6.1-1+deepin 
libinput10:amd64                              1.12.6.1-1+deepin
xserver-xorg-input-libinput                   0.28.2-2

**Comment by raspin0** [comment]
Are those logs meant to take  up so much space? 
![image](https://user-images.githubusercontent.com/39202451/173251114-d415c81a-ff9c-489c-9abd-fa8a0fee3577.png)

**Comment by kt286** [comment]
@raspin0 you can try this, it can solve some problems

https://github.com/better-dde/ppa/blob/master/dde-daemon_5.14.44-1%2Bkt286_amd64.deb

**Comment by BLumia** [comment]
I think I might have seen a similar report somewhere but I cannot find it. @justforlxz do you have any idea?

**Comment by kt286** [comment]
```c
static inline bool
check_event_type(struct libinput *libinput,
		 const char *function_name,
		 unsigned int type_in,
		 ...)
{
	bool rc = false;
	va_list args;
	unsigned int type_permitted;

	va_start(args, type_in);
	type_permitted = va_arg(args, unsigned int);

	while (type_permitted != (unsigned int)-1) {
		if (type_permitted == type_in) {
			rc = true;
			break;
		}
		type_permitted = va_arg(args, unsigned int);
	}

	va_end(args);

	if (!rc)
		log_bug_client(libinput,
			       "Invalid event type %d passed to %s()\n",
			       type_in, function_name);

	return rc;
}
```

400 LIBINPUT_EVENT_POINTER_MOTION
401 LIBINPUT_EVENT_POINTER_MOTION_ABSOLUTE

**Comment by kt286** [comment]
libinput error: client bug: start===============
libinput error: client bug: arg: 300
libinput error: client bug: end===============

libinput error: client bug: start===============
libinput error: client bug: arg: 300
libinput error: client bug: end===============

libinput error: client bug: start===============
libinput error: client bug: arg: 400
libinput error: client bug: arg: 401
libinput error: client bug: end===============

libinput error: client bug: start===============
libinput error: client bug: arg: 403
libinput error: client bug: Invalid event type LIBINPUT_EVENT_POINTER_MOTION_ABSOLUTE (401) passed to libinput_event_pointer_get_axis_source()
libinput error: client bug: end===============

libinput error: client bug: start===============
libinput error: client bug: arg: 403
libinput error: client bug: Invalid event type LIBINPUT_EVENT_POINTER_MOTION_ABSOLUTE (401) passed to libinput_event_pointer_get_axis_value()
libinput error: client bug: end===============

libinput error: client bug: start===============
libinput error: client bug: arg: 400
libinput error: client bug: arg: 401
libinput error: client bug: end===============

libinput error: client bug: start===============
libinput error: client bug: arg: 403
libinput error: client bug: Invalid event type LIBINPUT_EVENT_POINTER_MOTION_ABSOLUTE (401) passed to libinput_event_pointer_get_axis_source()
libinput error: client bug: end===============

libinput error: client bug: start===============
libinput error: client bug: arg: 403
libinput error: client bug: Invalid event type LIBINPUT_EVENT_POINTER_MOTION_ABSOLUTE (401) passed to libinput_event_pointer_get_axis_value()
libinput error: client bug: end===============

libinput error: client bug: start===============
libinput error: client bug: arg: 300
libinput error: client bug: end===============

libinput error: client bug: start===============
libinput error: client bug: arg: 300
libinput error: client bug: end===============

**Comment by kt286** [comment]
@raspin0 try this https://github.com/better-dde/ppa/blob/master/dde-daemon_5.14.44-1%2Bkt286%2B1_amd64.deb

**Comment by raspin0** [comment]
> @raspin0 try this https://github.com/better-dde/ppa/blob/master/dde-daemon_5.14.44-1%2Bkt286%2B2_amd64.deb

Fixed, thanks!

**Comment by SuperEffie** [comment]
Now our developers are already  to solve this bug. We will notify you when the problem is fixed. Looking forward to more feedback from you. Thank you~

**Comment by BLumia** [comment]
> Now our developers are already to solve this bug.

Could you link the related commit that fixes this issue? Thanks!

**Comment by shenmo7192** [comment]
> > Now our developers are already to solve this bug.
> 
> Could you link the related commit that fixes this issue? Thanks!

https://github.com/linuxdeepin/dde-daemon/commit/04c900d31414b3ea8f3f69da8e1542ce7b7d9e59

I believe it's here

**Comment by kt286** [comment]
> I believe it's here

No, this patch only reduces about 1/3 of the log output, the other 2/3 is caused by other bugs

**Comment by kt286** [comment]
this bug has been fixed at dde-daemon (5.14.58-1)
